templates/zerver/register.html

```diff
@@ -153,22 +153,16 @@
           <div class="help-box">
             <div id="org_type_blob_1" class="blob display-none">
               {% trans %}
-              Create a corporate organization if your users will be members of the
-              same company, where the privacy expectation is that the company's
-              organization administrators may need to have access to your users'
-              private message history. Zulip features that allow organization
-              administrators to take control of accounts or access private message
-              history for other users are only available to corporate organizations.
+              Organization admins will be able to take control of users'
+              accounts and access their private messages. Good for corporate
+              environments.
               {% endtrans %}
             </div>
             <div id="org_type_blob_2" class="blob">
               {% trans %}
-              Create a community organization for an open source project, social
-              group, or other community where the privacy expectation is that you
-              and other organization administrators won't have access to your users'
-              private message history. Zulip features that allow organization
-              administrators to take control of accounts or access private message
-              history for other users are disabled in community organizations.
+              Organization admins won't have access to take over other users'
```

Comment:
We should probably make both the definitions sound similar. So we can probably change

`won't have access to` -> `won't be able to`
`see their private messages` -> `access their private messages`

Just a recommendation though.